templates/pages/about.html

```diff
@@ -8,27 +8,59 @@
 
 
 	<div class="jumbotron">
-		<p>We, CodeCatz, are learning to code, beause we can and because we want to! </p>
-		<p>CodeCatz is a group of programming kittens and several slightly more experienced cats. Kittens want to learn coding, cats are here to help them. We started with Python and web applications, we'll continue with whatnots.</p>
-		<p> Who knows where we will end! </p>
+		<h2 class="about">About CodeCatz</h2>
+		<p>
```

Comment:
@PettraV: empty `<p>` tags can create a whole mess, it's not just a style issue. HTML does have a [`<br>` tag](http://www.w3schools.com/tags/tag_br.asp) that can be used on its own to create line breaks, but it's not recommended to manually hardcode line breaks. 

What you're looking for is the [CSS box model](http://www.w3schools.com/css/css_boxmodel.asp) and for bonus points even [line-height](http://www.w3schools.com/cssref/pr_dim_line-height.asp) :)